week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -12,12 +12,17 @@ let bottle = {
   volume: 0,
   fill: function() {
     // calling this function should make you bottles volume = 100;
+    bottle.volume=100;
   },
   drink: function() {
     // calling this function should decrease your bottles volume by 10;
+    bottle.volume=this.volume-10;

```

Comment:
This could be improved by doing this.

`this.volume = this.volume-10`

or even

`this.volume -= 10`